packages/progress-bar/test/progress-bar.test.js

```diff
@@ -27,6 +27,13 @@ describe('progress bar', () => {
 
     beforeEach(async () => {
       progress = fixtureSync('<vaadin-progress-bar></vaadin-progress-bar>');
+      fixtureSync(`
+        <style>
+          vaadin-progress-bar::part(value) {
+            transition: none;
```

Comment:
This is needed because `getBoundingClientRect()` returned `0` because of default transition.